Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 87 connect project content type to projects page

## Related ticket:
[SCRUM-87](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-87?atlOrigin=eyJpIjoiZWEwOWFkOGJlYzA0NGMzOGFmOTY0Y2UyZjYyODMxZjIiLCJwIjoiaiJ9)

## In this PR:
- It is hard to includes the projects field in Drupal so I just fetch it separately in the Projects page in the frontend
- Delete old Database files to avoid confusion

## Testing instruction
- Checkout branch
- Check the console on Projects page

[SCRUM-87]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-87?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
